test/tests/unit/wire_protocol_test.js

```diff
@@ -1,58 +1,55 @@
 'use strict';
 
-const expect = require('chai').expect;
+const chai = require('chai');
+const expect = chai.expect;
 const bson = require('bson');
+const sinon = require('sinon');
+const Pool = require('../../../lib/connection/pool.js');
 const wireProtocol2_6 = require('../../../lib/wireprotocol/2_6_support.js');
 const wireProtocol3_2 = require('../../../lib/wireprotocol/3_2_support.js');
 
-const MockPool = {};
-
-MockPool.write = function(cmd, options, callback) {
-  callback(null, cmd);
+const fake_topology = 'fake_topology';
```

Comment:
The topology is normally an object. I don't think it is worth it to stub the whole topology, but it should probably be an empty object `{}` instead of a string.